changelog/22.0/22.0.0/summary.md

```diff
@@ -167,3 +169,12 @@ All topology read calls _(`Get`, `GetVersion`, `List` and `ListDir`)_ now respec
 
 Building `vtadmin-web` now requires node >= v22.13.0 (LTS). Breaking changes from v20 to v22 can be found at https://nodejs.org/en/blog/release/v22.13.0 -- with no known issues that apply to VTAdmin.
 Full details on the node v20.12.2 release can be found at https://nodejs.org/en/blog/release/v22.13.1.
+
+## <a id="vtgate-flags"/>VTGate Flags
+
+### <a id="vtgate-init-tabletgw-wait-tablet-type"/> vtgate behavior for `--tablet_types_to_wait`
+Previously, if waiting for them took longer than the time specified in `--gateway_initial_tablet_timeout`, the `vtgate`
+would log a warning and  start serving. If queries were received by the `vtgate` for the tablet types that were not
+received on time, they would fail. Now, if waiting times out, a warning is logged, and the `vtgate` retries until it
```

Comment:
```suggestion
available would fail. In this release, this behavior has been changed so that if waiting times out, a warning is logged, and `vtgate` retries until it
```